core/src/commonMain/kotlin/ua/syt0r/kanji/presentation/screen/main/screen/writing_practice/WritingPracticeViewModel.kt

```diff
@@ -109,6 +113,12 @@ class WritingPracticeViewModel(
         viewModelScope.launch {
             preferencesRepository.setNoTranslationsLayoutEnabled(configuration.noTranslationsLayout)
             preferencesRepository.setLeftHandedModeEnabled(configuration.leftHandedMode)
+            preferencesRepository.setAltStrokeEvaluatorEnabled(configuration.altStrokeEvaluatorEnabled)
+
+            kanjiStrokeEvaluator = if(preferencesRepository.getAltStrokeEvaluatorEnabled())
```

Comment:
You don't need to get value from a repository here, just use `configuration.altStrokeEvaluatorEnabled`

Reply:
Done.